### PR TITLE
Prefer HTTPS URLs

### PR DESCRIPTION
**Pull request by nCastle1**
Only affects doc and default settings.

### CODE REVIEW

File: OfflineMapBook/OfflineMapBook_NetFramework/Properties/Settings.Designer.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ internal sealed partial class Settings : global::System.Configuration.Applicatio
         
         [global::System.Configuration.UserScopedSettingAttribute()]
         [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
-        [global::System.Configuration.DefaultSettingValueAttribute("http://runtime.maps.arcgis.com/sharing/rest")]
+        [global::System.Configuration.DefaultSettingValueAttribute("https://runtime.maps.arcgis.com/sharing/rest")]
```
Comment (esreli) [inline OfflineMapBook/OfflineMapBook_NetFramework/Properties/Settings.Designer.cs:52]: Can you confirm the app operates as expected after this change?

File: OfflineMapBook/OfflineMapBook_NetFramework/Properties/Settings.settings [hunk 2]
```diff
@@ -9,7 +9,7 @@
       <Value Profile="(Default)">OfflineMapbookTest.mmpk</Value>
     </Setting>
     <Setting Name="PortalUri" Type="System.String" Scope="User">
-      <Value Profile="(Default)">http://runtime.maps.arcgis.com/sharing/rest</Value>
+      <Value Profile="(Default)">https://runtime.maps.arcgis.com/sharing/rest</Value>
```
Comment (esreli) [inline OfflineMapBook/OfflineMapBook_NetFramework/Properties/Settings.settings:12]: Can you confirm the app operates as expected after this change?

File: OfflineMapBook/OfflineMapBook_NetFramework/app.config [hunk 3]
```diff
@@ -22,7 +22,7 @@
         <value>OfflineMapbookTest.mmpk</value>
       </setting>
       <setting name="PortalUri" serializeAs="String">
-        <value>http://runtime.maps.arcgis.com/sharing/rest</value>
+        <value>https://runtime.maps.arcgis.com/sharing/rest</value>
```
Comment (esreli) [inline OfflineMapBook/OfflineMapBook_NetFramework/app.config:25]: Can you confirm the app operates as expected after this change?

**Review by esreli — APPROVED** [review]
This LGTM. My only remaining question is, does the app still perform as expected with changes to hard coded URIs in settings? My approval here is contingent that there are no problems with the modified Settings URIs.

**Review by esreli — CHANGES_REQUESTED** [review]
I just noticed the base branch is to `main`. Please cut a patch release.

**Comment by mikewilburn** [comment]
I imagine we're good to go so long as we amend the RELEASE.md with this PR and then create a 1.0.6 release?
That way the target branch can remain on `main` and the "from" branch name won't need to change either. Do you agree @esreli?

**Comment by esreli** [comment]
https://github.com/Esri/mapbook-wpf/pull/44#issuecomment-718931026

That's right 👍

File: OfflineMapBook/OfflineMapBook_NetFramework/Properties/Settings.Designer.cs (hunk 1 above)
Comment (nCastle1) [reply OfflineMapBook/OfflineMapBook_NetFramework/Properties/Settings.Designer.cs:52]: I confirmed that the app works as it should after this change

**Pull request merged**